csharp/extractor/Semmle.Extraction.CSharp.DependencyFetching/DependencyManager.Nuget.cs

```diff
@@ -479,27 +579,63 @@ private IEnumerable<string> GetFeeds(string nugetConfig)
                     continue;
                 }
 
-                yield return url;
+                if (!string.IsNullOrWhiteSpace(url))
+                {
+                    yield return url;
+                }
             }
         }
 
-        private HashSet<string> GetAllFeeds(List<FileInfo> allFiles)
+        private (HashSet<string>, HashSet<string>) GetAllFeeds(List<FileInfo> allFiles)
```

Comment:
Even though this method is private, maybe add some description of the return since it is a tuple of hashsets of the same type (to avoid confusing them)